src/pages/auth/loginPage.tsx

```
@@ -0,0 +1,158 @@
+const initialValues: LoginPayload = {
+  email: "customer123@gmail.com",
+  password: "1234",
+};
+
+const loginSchema = Yup.object().shape({
+  email: Yup.string().required().email(),
+  password: Yup.string().required(),
+});
+
+const LoginPage: React.FC = () => {
+  const dispatch = useAppDispatch();
```

ESLint

[error] 'dispatch' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)